ui/lib/pki/addon/templates/tidy.hbs

```diff
@@ -0,0 +1 @@
+{{outlet}}
```

Comment:
should we remove this file? since we have a `tidy/index.hbs` we put the content in there?

Reply:
the router wasn't going to `tidy.auto.configure` correctly until I added this 🤔